src/git-data.ts

```diff
@@ -100,7 +100,7 @@ export class GitData {
             }
 
             if (gitRemote.startsWith("http")) {
-                gitRemoteMatch = /(?<schema>https?):\/\/(?:(\w+):([\w-]+)@)?(?<host>[^/:]+):?(?<port>\d+)?\/(?<group>\S+)\/(?<project>\S+)\.git/.exec(gitRemote); // regexr.com/7ve8l
+                gitRemoteMatch = /(?<schema>https?):\/\/(?:([^:]+):([^@]+)@)?(?<host>[^/:]+):?(?<port>\d+)?\/(?<group>\S+)\/(?<project>\S+)\.git/.exec(gitRemote); // regexr.com/7ve8l
```

Comment:
```suggestion
                gitRemoteMatch = /(?<schema>https?):\/\/(?:([^:]+):([^@\s]+)@)?(?<host>[^/:]+):?(?<port>\d+)?\/(?<group>\S+)\/(?<project>\S+)\.git/(?<project>\S+)\.git/.exec(gitRemote); // regexr.com/7ve8l
```

so that it looks nicer on the regexr.com